### PR TITLE
media-types: Define layer media types suffix '+zstd'

### PR DESCRIPTION
**Pull request by giuseppe**
Closes: https://github.com/opencontainers/image-spec/issues/787

Signed-off-by: Giuseppe Scrivano <gscrivan@redhat.com>

### CODE REVIEW

File: media-types.md [hunk 1]
```diff
@@ -50,6 +52,12 @@ This section shows where the OCI Image Specification is compatible with formats
 
 - [application/vnd.docker.image.rootfs.diff.tar.gzip](https://github.com/docker/docker/blob/master/image/spec/v1.md#creating-an-image-filesystem-changeset)
 
+### application/vnd.oci.image.layer.v1.tar+zstd
+
+**Interchangeable and fully compatible mime-types**
+
+- [application/vnd.docker.image.rootfs.diff.tar.zstd](https://github.com/docker/docker/blob/master/image/spec/v1.md#creating-an-image-filesystem-changeset)
```
Comment (AkihiroSuda) [inline media-types.md]: this does not exist

File: media-types.md [hunk 2]
```diff
@@ -9,8 +9,10 @@ The following media types identify the formats described here and their referenc
 - `application/vnd.oci.image.config.v1+json`: [Image config](config.md)
 - `application/vnd.oci.image.layer.v1.tar`: ["Layer", as a tar archive](layer.md)
 - `application/vnd.oci.image.layer.v1.tar+gzip`: ["Layer", as a tar archive](layer.md#gzip-media-types) compressed with [gzip][rfc1952]
+- `application/vnd.oci.image.layer.v1.tar+zstd`: ["Layer", as a tar archive](layer.md#zstd-media-types) compressed with [zstd][rfc8478]
 - `application/vnd.oci.image.layer.nondistributable.v1.tar`: ["Layer", as a tar archive with distribution restrictions](layer.md#non-distributable-layers)
 - `application/vnd.oci.image.layer.nondistributable.v1.tar+gzip`: ["Layer", as a tar archive with distribution restrictions](layer.md#gzip-media-types) compressed with [gzip][rfc1952]
+- `application/vnd.oci.image.layer.nondistributable.v1.tar+zstd`: ["Layer", as a tar archive with distribution restrictions](layer.md#zstd-media-types) compressed with [zstd][rfc8478]
```
Comment (AkihiroSuda) [inline media-types.md]: needs link for rfc8478

**Comment by vbatts** [comment]
LGTM

**Comment by giuseppe** [comment]
@AkihiroSuda thanks for the review.  Pushed a new version

**Comment by rhatdan** [comment]
@brendandburns @cyphar @jbouzane @jstarks @philips @stevvooe @xiekeyang PTAL

@giuseppe Any idea how to fix travis?

**Comment by giuseppe** [comment]
> @giuseppe Any idea how to fix travis?

I've opened another PR to address the error raised by "make lint": https://github.com/opencontainers/image-spec/pull/789

**Comment by giuseppe** [comment]
rebased on top of the fix for travis

**Comment by jonboulle** [comment]
LGTM

**Comment by rhatdan** [comment]
@vbatts PTAL

**Comment by vbatts** [comment]
LGTM

**Pull request merged**